arq/cli.py

```diff
@@ -28,15 +30,24 @@
 @click.option('--check', is_flag=True, help=health_check_help)
 @click.option('--watch', type=click.Path(exists=True, dir_okay=True, file_okay=False), help=watch_help)
 @click.option('-v', '--verbose', is_flag=True, help=verbose_help)
-def cli(*, worker_settings: str, burst: bool, check: bool, watch: str, verbose: bool) -> None:
+@click.option('--custom-log-dict', type=str, help=logdict_help)
+def cli(*, worker_settings: str, burst: bool, check: bool, watch: str, verbose: bool, custom_log_dict: str) -> None:
     """
     Job queues in python with asyncio and redis.
 
     CLI to run the arq worker.
     """
     sys.path.append(os.getcwd())
     worker_settings_ = cast('WorkerSettingsType', import_string(worker_settings))
-    logging.config.dictConfig(default_log_config(verbose))
+    if custom_log_dict:
+        try:
+            config_path, config_dict = custom_log_dict.rsplit('.', maxsplit=1)
+            log_config = getattr(importlib.import_module(config_path), config_dict)
+        except (TypeError, AttributeError):
+            log_config = default_log_config(verbose)
```

Comment:
we should log (probably just print to sterr is easiest) a warning in this case with details of what went wrong.

Or we could just raise an error.

Reply:
Raising an ImportError from `import_string` seems like it would work okay - I'll do that.